### ISSUE TITLE
getReports errors out

### ISSUE THREAD

**Issue by SystemCrashed** [ChrisMichaelPerezSantiago/covid19#12]
Used to be able to use `.getReports()` but no longer can. I have changed nothing. It's just, stopped working. Any ideas?
```
2|client  | (node:141183) UnhandledPromiseRejectionWarning: TypeError: Property description must be an object: undefined
2|client  |     at Function.defineProperty (<anonymous>)
2|client  |     at renameKey (/root/Says/node_modules/covid19-api/src/api/utils/utils.js:3:12)
2|client  |     at /root/Says/node_modules/covid19-api/src/api/api.js:85:26
2|client  |     at Array.forEach (<anonymous>)
2|client  |     at /root/Says/node_modules/covid19-api/src/api/api.js:85:9
2|client  |     at Array.map (<anonymous>)
2|client  |     at Object.reports (/root/Says/node_modules/covid19-api/src/api/api.js:83:17)
2|client  |     at async Promise.all (index 0)
2|client  |     at async PluginManager.getReports (/root/Says/node_modules/covid19-api/src/api/PluginManager.js:7:17)
2|client  |     at async Timeout.eval [as _onTimeout] (eval at exports.run (/root/Says/commands/eval.js:11:22), <anonymous>:1:38)
2|client  | (node:141183) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 9)
```

**Comment by Abinash-bapu123** [comment]
[https://covid19-server.chrismichael.now.sh/api/v1/AllReports](url)
### This API is not working anymore

**Comment by SystemCrashed** [comment]
#13 Someone else experiencing the same issue.

**Comment by ChrisMichaelPerezSantiago** [comment]
Hi @SystemCrashed 

Problem solved #11
